nautobot/docs/installation/services.md

```diff
@@ -144,6 +147,9 @@ WantedBy=multi-user.target
 +/- 1.1.0
     Prior to version 1.1.0, Nautobot utilized RQ as the primary background task worker. As of Nautobot 1.1.0, RQ is now *deprecated* and has been replaced with Celery. RQ can still be used by plugins for now, but will be removed in a future release. Please [migrate your deployment to utilize Celery as documented below](#migrating-to-celery-from-rq).
```

Comment:
Is this still accurate or should it be changed to remove `RQ can still be used by plugins for now, but will be removed in a future release`

Reply:
Good call, I'll revise this.